### ISSUE TITLE
[DISCUSSION] Add Favourite IDE / IDE-Flair as an Attribute

### ISSUE THREAD

**Issue by Novalis15** [benawad/vsinder#53]
I know vsinder is mainly done for vscode users (hence the name i guess) and therefore it seems redundant to add an IDE-Flair attribute. However, vsinder users on IOS or Android are not bound to vscode specifically.

What do you think?

**Comment by John-Trager** [comment]
I like the idea and feel it could add more depth to the whole experience!

**Comment by gernei09** [comment]
Absolutely. You already have tons of profiles out there that are specifying their fav editor anyhow.

**Comment by unDeadHerbs** [comment]
Especially with #157 on the way.
